pkg/debug/spec.sh

```diff
@@ -393,62 +454,16 @@ __EOT__
    fi
 done
 
-#enumerate cellular modems
-ID="0"
-for USBDEV in /sys/class/usbmisc/*; do
-    get_modem_protocol "$USBDEV" >/dev/null || continue
-    IFNAME="$(get_modem_ifname "$USBDEV")"
-    MODEMS="${MODEMS}${IFNAME}\n" # skip during NIC enumeration
-    USB_ADDR="$(get_modem_usbaddr "$USBDEV")"
-    PCI_ADDR="$(get_modem_pciaddr "$USBDEV")"
-    GROUP=$(get_assignmentgroup "${IFNAME}" "${PCI_ADDR}")
-    cat <<__EOT__
-    ${COMMA}
-    {
-      "ztype": 6,
-      "logicallabel": "modem${ID}",
-      "phylabel": "modem${ID}",
-      "assigngrp": "${GROUP}",
-      "phyaddrs": {
-        "PciLong": "${PCI_ADDR}",
-        "UsbAddr": "${USB_ADDR}"
-      },
-      "cost": 10,
-__EOT__
-    if [ -n "$verbose" ] && [ -x "$(command -v picocom)" ]; then
-        AT_PORT="$(get_modem_atport "$USB_ADDR")"
-        if [ -n "$AT_PORT" ]; then
-            # Close any previous communication at this AT port.
-            send_at_command "+++" "${AT_PORT}"
-            # Return modem identification.
-            # Use sed to remove lines containing only whitespace.
-            OUTPUT="$(send_at_command "ATI" "${AT_PORT}" | sed '/^\s*$/d')"
-            if echo "$OUTPUT" | grep -qv "ERROR"; then
-                # Remove control commands.
-                OUTPUT="$(echo "$OUTPUT" | sed '/^\s*ATI\s*$/d; /^\s*OK\s*$/d' | tr -d '\r')"
-                # Keep at most two lines and join them by semi-colon.
-                OUTPUT="$(echo "$OUTPUT" | head -n 2 | sed ':a;N;$!ba;s/\n/; /g')"
-                cat <<__EOT__
-      "description": "$OUTPUT",
-__EOT__
-            fi
-        fi
-    fi
-    cat <<__EOT__
-      "usagePolicy": {}
-__EOT__
-    COMMA="},"
-    ID=$(( ${ID:-0} + 1 ))
-done
-
-#enumerate NICs (except for cellular modems)
+#enumerate NICs (ignoring USB devices)
 for ETH in /sys/class/net/*; do
    LABEL=$(echo "$ETH" | sed -e 's#/sys/class/net/##' -e 's#^k##')
+   # Does $LABEL start with wlan or wwan? Change ztype and cost
```

Comment:
Seems like an obsolete and no longer appropriate comment

Reply:
What about devices that are not connected via usb?

Reply:
We do not support cellular modems connected over PCI yet.
As for WiFi, `if [ -d "$ETH/wireless" ]` used below should work regardless of the bus type.